website_twitter_no_ext_links/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_twitter_feed_user,twitter.feed.user,model_twitter_feed,base.group_user,1,1,1,1
+access_twitter_theme_user,twitter.theme.user,model_twitter_theme,base.group_user,1,1,1,1
+access_twitter_user_user,twitter.user.user,model_twitter_user,base.group_user,1,1,1,1
+access_twitter_tweet_user,twitter.tweet.user,model_twitter_tweet,base.group_user,1,1,1,1
```

Comment:
shouldn't they also be readable by everyone? (that is, the group_id field empty)
otherwise, you'll have to sudo all calls from the website, because there, typically there's only the website user